src/test/java/org/takes/rs/RsWithTypeTest.java

```diff
@@ -52,11 +57,41 @@ public void replaceTypeToResponse() throws IOException {
                 )
             ).print(),
             Matchers.equalTo(
-                Joiner.on("\r\n").join(
-                    "HTTP/1.1 200 OK",
-                    String.format("Content-Type: %s", type),
+                Joiner.on(CRLF).join(
+                        "HTTP/1.1 200 OK",
+                        String.format("Content-Type: %s", type),
+                        "",
+                        ""
+                )
+            )
+        );
+    }
+
+    /**
+     * RsWithType can not replace response code.
+     * @throws java.io.IOException If a problem occurs.
+     */
+    @Test
+    public void notReplacesResponseCode() throws IOException {
```

Comment:
@ekondrashev Let's name this method `cannotReplaceResponseCode`. Also, let's declare `Exception` being thrown here, as per the practice [described here](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names).